src/scripts/twitter-handles.data.js

```diff
@@ -0,0 +1,39 @@
+const { en } = require('../lang/en')
+const { getTwitterHandles, saveTwitterHandles } = require('../TweetDbService')
+
+/* eslint-disable no-undef */
+const { TECH } = en.category
+const handles = [
+	{
+		name: 'Shrjan',
+		handle: '@itsmeshrijan',
+		category: TECH,
+		weight: 20
+	}
+]
+
+const insertTwitterHandles = async () => {
```

Comment:
@shrijan-siris for now, you can insert using simple mongo statements like in `source-data.js`